site/it/tutorials/keras/regression.ipynb

```diff
@@ -0,0 +1,900 @@
+      },
+      "outputs": [],
+      "source": [
+        "# Use seaborn for pairplot\n",
+        "!pip install seaborn\n",
+        "\n",
+        "# Use some functions from tensorflow_docs\n",
+        "!pip install git+https://github.com/tensorflow/docs"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "1rRo8oNqZ-Rj"
+      },
+      "outputs": [],
+      "source": [
+        "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+        "\n",
+        "import pathlib\n",
+        "\n",
+        "import matplotlib.pyplot as plt\n",
+        "import numpy as np\n",
+        "import pandas as pd\n",
+        "import seaborn as sns"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "9xQKvCJ85kCQ"
+      },
+      "outputs": [],
+      "source": [
+        "try:\n",
+        "  # %tensorflow_version only exists in Colab.\n",
+        "  %tensorflow_version 2.x\n",
+        "except Exception:\n",
+        "  pass\n",
+        "import tensorflow as tf\n",
+        "\n",
+        "from tensorflow import keras\n",
+        "from tensorflow.keras import layers\n",
+        "\n",
+        "print(tf.__version__)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "Qz4HfsgRQUiV"
+      },
+      "outputs": [],
+      "source": [
+        "import tensorflow_docs as tfdocs\n",
+        "import tensorflow_docs.plots\n",
+        "import tensorflow_docs.modeling"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "F_72b0LCNbjx"
+      },
+      "source": [
+        "## Il dataset Auto MPG\n",
+        "\n",
+        "Il dataset è disponibile su [UCI Machine Learning Repository](https://archive.ics.uci.edu/ml/).\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "gFh9ne3FZ-On"
+      },
+      "source": [
+        "### Otteniamo i dati\n",
+        "Per prima cosa, scarichiamo il dataset."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "p9kxxgzvzlyz"
+      },
+      "outputs": [],
+      "source": [
+        "dataset_path = keras.utils.get_file(\"auto-mpg.data\", \"http://archive.ics.uci.edu/ml/machine-learning-databases/auto-mpg/auto-mpg.data\")\n",
+        "dataset_path"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "nslsRLh7Zss4"
+      },
+      "source": [
+        "Importiamolo usando i panda"
```

Comment:
I was expecting here just to have "pandas", as in the library name instead of "i panda", which by reading it I feel like it refers to the animal panda. A full Italian sentence could be "Importiamolo usando la libreria pandas", it gives the idea of the meaning but doesn't full correspond to the English sentence "Import it using pandas". What do you think about this point?